src/main/java/edu/project2/MazePrinter.java

```diff
@@ -0,0 +1,22 @@
+package edu.project2;
+
+import java.util.Set;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class MazePrinter {
+    private final static Logger LOGGER = LogManager.getLogger();
+
+    private final MazeRenderer renderer;
```

Comment:
Хотелось бы здесь видеть зависимость от абстракции (буква D из принципов SOLID):
```suggestion
    private final Renderer renderer;
    
    public MazePrinter(Renderer renderer) {
        this.renderer = renderer;
    }
```
для того, чтобы можно было передать сюда любую другую реализацию Renderer (например, если ты захочешь отрисовывать лабиринт в другом стиле).